jupyterhub_traefik_proxy/kv_proxy.py

```diff
@@ -32,50 +32,29 @@
 class TKvProxy(TraefikProxy):
     """
     JupyterHub Proxy implementation using traefik and a key-value store.
-    Custom proxy implementations based on trafik and a key-value store
+
+    Custom proxy implementations based on traefik and a key-value store
     can sublass :class:`TKvProxy`.
     """
 
-    kv_client = Any()
```

Comment:
I removed these generic traits from KVProxy which weren't actually generic - these are unused by the base class and not actually general, even though our two subclasses have sufficiently similar setups.

I traded them for more descriptive and specific `etcd_username`, etc. config on the specific implementation classes, since these should be implementation-specific details, not generic features.